packages/fx-core/resource/package.nls.json

```diff
@@ -916,14 +930,20 @@
   "depChecker.portableFuncNodeNotMatched": "Your own Node.js(@NodeVersion) is incompatible with Teams Toolkit Azure Functions Core Tools(@FuncVersion). Please install the supported Node.js version(@SupportedVersions).",
   "depChecker.startInstallNgrok": "Downloading and installing @NameVersion.",
   "depChecker.finishInstallNgrok": "Successfully installed @NameVersion.",
+  "depChecker.finishInstallBicep": "Successfully installed @NameVersion.",
   "depChecker.needInstallNgrok": "You must have @NameVersion installed to debug your local bot.",
   "depChecker.failToInstallNgrok": "Failed to install @NameVersion. Install @NameVersion manually.",
   "depChecker.failToValidateNgrok": "Failed to validate @NameVersion after installation.",
   "depChecker.downloadDotnet": "Downloading and installing the portable version of @NameVersion, which will be installed to @InstallDir and will not affect your environment.",
+  "depChecker.downloadBicep": "Downloading and installing the portable version of @NameVersion, which will be installed to @InstallDir and will not affect your environment.",
   "depChecker.finishInstallDotnet": "Successfully installed @NameVersion.",
   "depChecker.useGlobalDotnet": "Using dotnet from PATH:",
   "depChecker.dotnetInstallStderr": "dotnet-install command failed without error exit code but with non-empty standard error.",
   "depChecker.dotnetInstallErrorCode": "dotnet-install command failed.",
+  "depChecker.failToInstallBicepOutputVSC": "Failed to install @NameVersion. please read this wiki(@HelpLink) to install @NameVersion manually and restart Visual Studio Code.",
```

Comment:
The "@NameVersion" and "@HelpLink" fields are hard to understand. Translation team might get confused whether they should translate this. Suggestions are:
1> Use %s to replace the fields here and update related codes to: getLocalizedString(Message.xxx, <name_version>, <help_link>)
2> Add comments here to indicate this doesn't need to be translated